.php_cs.dist

```diff
@@ -0,0 +1,40 @@
+<?php
+
+$config = PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@PSR2' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'declare_strict_types' => false,
+        'function_to_constant' => ['functions' => ['php_sapi_name']],
+        'concat_space' => ['spacing'=>'one'],
+        'php_unit_test_case_static_method_calls' => ['call_type' => 'self'],
+        'php_unit_no_expectation_annotation' => ['target' => 'newest'],
+        'php_unit_expectation' => ['target' => 'newest'],
+        'php_unit_dedicate_assert_internal_type' => ['target' => 'newest'],
+        'no_empty_phpdoc' => true,
+        'no_extra_blank_lines' => true,
+        'ordered_imports' => true,
+        'no_superfluous_phpdoc_tags' => ['allow_mixed' => true],
+        'no_blank_lines_after_phpdoc' => true,
+        'no_unused_imports' => true,
+        'phpdoc_trim_consecutive_blank_line_separation' => true,
+        'phpdoc_trim' => true,
+        'phpdoc_scalar' => true,
+        'phpdoc_types' => true,
+        'phpdoc_var_without_name' => true,
+        'phpdoc_single_line_var_spacing' => true,
+        'phpdoc_separation' => true,
+        'phpdoc_align' => ['align'=>'vertical'],
+        'visibility_required' => true,
+        // 'native_function_invocation' => true,
```

Comment:
please consider adding these:

- https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:php_unit_mock_short_will_return
- https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:final_internal_class
- https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:standardize_not_equals 
- https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:single_trait_insert_per_statement
- https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:return_type_declaration
- https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:is_null
- https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:short_scalar_cast
- https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:no_unset_on_property

Some of these might not seem to have value today.. but by adding them now we will save more time from future reviews

Check here if we need something more.. 
https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16

Reply:
Copy-pasted from Guzzle and I want to keep in sync with it.

Reply:
At some point one of the two will need to be updated with something that the other will not have and we might go to the other and update it. 

It would be nice if we could just come up with a set of final rules.. 

I understand if you don't want to do it now and wanna have a separate discussion about that.. if so.. you can resolve this and we can check it later...